src/appmixer/naxai/auth.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+module.exports = {
+
+    type: 'apiKey',
+
+    definition: {
+
+        auth: {
+            clientId: {
+                type: 'text',
+                name: 'Client ID',
+                tooltip: 'Provide your client ID.'
+            },
+
+            clientSecret: {
+                type: 'password',
+                name: 'Client Secret',
+                tooltip: 'Provide your client secret.'
+            }
+        },
+
+        accountNameFromProfileInfo: 'clientId',
```

Comment:
@jirihofman Please add another field for the account name in the authentication screen keeping clientId as a fallback name.

Reply:
What would be the benefit of it? clientId identifies exactly the api credentials that are used by the component.

Reply:
Users can add custom account names for easier association with client IDs which are hard to rember.

Reply:
But it can also introduce more problems when user names several `clientId`s in the same way, eg. "My account". And we don't have any way to do some unique check here. I think it is fine as it is. It's the same as with any other `apiKey` authentication where user provides string like these.